### DISCUSSION TITLE
Using a new version of kustomize on argocd

### DISCUSSION THREAD

**Discussion by abdiali9617** [argoproj/argo-cd#6394] [Q&A]
Is there a way to use a newer version of kustomize with argocd 2 (specifically v4.1.3)?

**Comment by jannfis** [comment]
Have you checked out https://argo-cd.readthedocs.io/en/stable/user-guide/kustomize/#custom-kustomize-versions ?

It may work to use 4.x with Argo CD 2.0, if the CLI had no backward incompatible changes.

That being said, Argo CD v2.1 will come bundled with Kustomize v4 (and latest builds already have it as well).
